src/plugins/data/public/query/query_string/language_service/types.ts

```diff
@@ -51,4 +51,5 @@ export interface LanguageConfig {
   showDocLinks?: boolean;
   editorSupportedAppNames?: string[];
   supportedAppNames?: string[];
+  noTimeField?: boolean;
```

Comment:
nit: perhaps it should be apart of the fields? as `aggregatable` as there is no field that can get any aggregations for

Reply:
if not that then perhaps something like disableDatePicker

since noTimeField might be a little confusion for it's functionality

Reply:
Just to clarify, is `aggregatable` only referring to `date` fields? Is there anything else that could mean aggregatable for SQL? Putting as `disableDatePicker` until I have a better understanding of that.

Reply:
disabledatepicker is cool for me. or even if we want to keep in line with `showDocLinks` it could be `showDatePicker`